### PR TITLE
Fixed ratemyprofessor link

### PR DESCRIPTION
**Pull request by esoccoli**

### CODE REVIEW

File: api/rmp.php [hunk 1]
```diff
@@ -11,5 +11,5 @@
 curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
 curl_setopt($curl, CURLOPT_HEADER, false);
 $name = explode('/', $_SERVER['REQUEST_URI'])[2];
-curl_setopt($curl, CURLOPT_URL, "http://www.ratemyprofessors.com/SelectTeacher.jsp?searchName=" . $name . "&search_submit1=Search&sid=807");
+curl_setopt($curl, CURLOPT_URL, "http://www.ratemyprofessors.com/SelectTeacher.jsp?searchName= . $name . "&search_submit1=Search&sid=807");
```
Comment (Mstrodl) [inline api/rmp.php:14]: I really don't think this does what you think it does

**Review by Mstrodl — CHANGES_REQUESTED** [review]

**Comment by jabbate19** [comment]
Fixes #324

**Comment by nsp7786** [comment]
Based on #324 , you'd want a link like this: `https://www.ratemyprofessors.com/search/professors?q=${name}&sid=${schoolId}` (RIT's school id is `U2Nob29sLTgwNw==`)

**Comment by MoralCode** [comment]
As the author of #324 and based on the line under review above, it looks like the line should be
`curl_setopt($curl, CURLOPT_URL, "https://www.ratemyprofessors.com/search/professors?q=" . $name . "&sid=U2Nob29sLTgwNw==");`

Would be awesome to see this get fixed in ScheduleMaker!

**Comment by Mstrodl** [comment]
Obsolete, and wasn't the right file anyways.

**Comment by MoralCode** [comment]
ah cool, thanks for fixing this! (commit: https://github.com/ComputerScienceHouse/schedulemaker/commit/77ad61b183ad700c95f511cf96d6954af89b478c for anyone looking to see how it was done)

If this indeed "wasn't the right file anyways", then it seems as though the core issue (#324) is still not resolved

**Pull request closed without merging**